src/tilemaker/entrypoint.sh

```diff
@@ -7,6 +7,8 @@ source /opt/pfb/tilemaker/scripts/utils.sh
 
 export TL_MIN_ZOOM="${TL_MIN_ZOOM:-8}"
 export TL_MAX_ZOOM="${TL_MAX_ZOOM:-18}"
+export TL_NUM_PARTS="${TL_NUM_PARTS:-4}"
```

Comment:
Yeah I think one-way partitioning to keep things simple at this point is fine. Seems like 4vcpus is a good middle ground?